lib/stdlib/doc/src/gen_event.xml

```diff
@@ -142,8 +142,12 @@ gen_event:stop             ----->  Module:terminate/2
     </datatype>
 
     <datatype>
-      <name name="emgr_ref"/>
-    </datatype>    
+      <name name="process_ref"/>
```

Comment:
emgr is not a clear name I think, I guess it stands for event manager. But in this particular place I am not sure if it is obvious if we should use the generalized type or if we should have a type for this abstraction?!  I think the same goes for gen_server and supervisor module. I will make a comment in those place too, so that we can have comments from the @erlang/team-ps

Reply:
Or maybe it is the case that we should not have new types with the same name but let the documentation refer to 
proc_lib:process_name and so on!

Reply:
The point on making aliases an exporting the aliases is to provide a type abstraction over it, so that theoretically, in a common type system, `gen_event:process_ref()` is not the same as other type that happens to match its internal representation. That said, the current type checker (Dialyzer) will expand the types to its constituents, so for Dialyzer `gen_event:process_ref()` and `proc_lib:process_ref()` are the same. 
(AFAIK, Dialyzer is a structural type system and will establish the equality of types as mentioned above)

Reply:
Regarding the generalisation of the type, I know think you mean that `emgr_ref()` is more concrete than `process_ref()`.
I would like to favour uniformity, so that we can create a "type" understanding without having to know that an `emgr_ref()` is equivalent to what is known in other modules as `process_ref()`